src/components/RecentOperationsCards/RecentOperationsCards.tsx

```diff
@@ -48,14 +43,14 @@ export const RecentOperationDashboardCard: FC<IRecentOperationDashboardCard> = (
             </div>
             <p style={{ color: isExpense ? "#FF2D55" : "#80D667" }}
                 className={classes.amount}>
-                {`${isExpense ? "-" : "+"}$${numberWithCommas(item.amount)}`}</p>
+                {`${isExpense ? "-" : "+"}${currency}${numberWithCommas(item.amount)}`}</p>
         </li>
     )
 }
 
 
 export const RecentOperationGroupCard: FC<IRecentOperationGroupCard> = ({ categoryColor, categoryTitle, time, amount, member, type }) => {
-
+    const currency = useAppSelector(state => state.persistedCurrencySlice.currency)
```

Comment:
useAppSelector<Here type of store>(state => state.persistedCurrencySlice.currency)